foundations/introduction/motivation_and_mindset.md

```diff
@@ -176,7 +176,7 @@ This section contains helpful links to related content. It isn't required, so co
 - [Becoming a TOP Success Story](https://dev.to/theodinproject/becoming-a-top-success-story-mindset-3dp2)
 - [Managing inspiration and motivation](https://markmanson.net/do-something)
 - [Learning to code when it gets dark](https://medium.freecodecamp.org/learning-to-code-when-it-gets-dark-e485edfb58fd#.yjh0fehje)
-- While thinking and research time is a bigger bottleneck than raw typing speed, if you'd like to improve your typing skills then some excellent options include [typing.com](https://www.typing.com/), [TypingClub.com](https://www.typingclub.com) and [Monkeytype](https://monkeytype.com/).
+- While thinking and research time is a bigger bottleneck than raw typing speed, if you'd like to improve your typing skills then some excellent options include [typing.com](https://www.typing.com/), [TypingClub.com](https://www.typingclub.com) and [Monkeytype](https://monkeytype.com/). [Keybr](https://www.keybr.com) even has a "Source Code" setting specifically designed to help you practice commonly used programming keys!
```

Comment:
This is getting quite cluttered 😅. If you had to replace one of the existing links with Keybr which would it be @Lanerbop?

Reply:
Typing.com is a solid resource, and Monkeytype is beloved by the typing community due to its customization features. Typingclub.com, on the other hand, seems more geared toward small children, so I would suggest replacing that one with Keybr!

Reply:
Thanks @Lanerbop! would you mind doing that in this PR please?

Reply:
Done!